### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by dreis2211**
Hi,

this PR fixes a couple of typos. I should note that the particular sentence in `spring-boot-features` still doesn't make a lot of sense to me even after the fix, but as I'm not a native speaker I wasn't sure if it might be correct.

Cheers,
Christoph

### CODE REVIEW

**Comment by snicoll** [comment]
> I should note that the particular sentence in spring-boot-features still doesn't make a lot of sense to me even after the fix

I wrote that and I agree the phrasing is a bit odd. I'll try to improve it as part of the polish.

File: spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/runner/AbstractApplicationContextRunner.java [hunk 1]
```diff
@@ -179,8 +179,8 @@ public SELF withAllowBeanDefinitionOverriding(boolean allowBeanDefinitionOverrid
 	}
 
 	/**
-	 * Add a {@link ApplicationContextInitializer} to be
-	 * calWebMvcEndpointChildContextConfigurationTestsled when the context is created.
```
Comment (snicoll) [inline spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/runner/AbstractApplicationContextRunner.java]: oh dear 🙈

**Comment by snicoll** [comment]
Thanks again @dreis2211. I've polished that sentence in https://github.com/spring-projects/spring-boot/commit/8154e672ddc91e07e8cbd77bc8511b263d94a4ce. Let me know if that still reads odd.

File: spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/runner/AbstractApplicationContextRunner.java (hunk 1 above)
Comment (dreis2211) [reply spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/runner/AbstractApplicationContextRunner.java]: I liked that one 🤣

**Comment by dreis2211** [comment]
Isn't that a different sentence? You polished the one about `DurationUnits` and not `DataSizeUnits`. Nonetheless, it reads better ;-)

**Comment by snicoll** [comment]
It is. Looks like that pattern was applied for `DataSize` as well. I've polished the other one in https://github.com/spring-projects/spring-boot/commit/49ab5de91c8ee867ea87591a4aa0137e90f1d551

**Pull request closed without merging**